app/src/main/java/com/sun/android/bai7_Order/OrderActivity.kt

```diff
@@ -0,0 +1,76 @@
+package com.sun.android.bai7_Order
+
+import android.content.Intent
+import android.os.Bundle
+
+import android.view.View
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import android.widget.RadioButton
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.sun.android.R
+import com.sun.android.databinding.ActivityOrderBinding
+
+
+class OrderActivity : AppCompatActivity(), AdapterView.OnItemSelectedListener {
+    val binding by lazy { ActivityOrderBinding.inflate(layoutInflater) }
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        val message: String? = intent.getStringExtra("MESSAGE")
+        binding.orderTextview.text = message
+        val spinner = binding.labelSpinner
+        if (spinner != null) {
+            spinner.setOnItemSelectedListener(this)
+        }
+        val adapter: ArrayAdapter<CharSequence> = ArrayAdapter.createFromResource(
+            this,
+            R.array.labels_array,
+            android.R.layout.simple_spinner_item
+        )
+        adapter.setDropDownViewResource(
+            android.R.layout.simple_spinner_dropdown_item
+        )
+        if (spinner != null) {
+            spinner.adapter = adapter;
+        }
+    }
+
+    override fun onNothingSelected(p0: AdapterView<*>?) {
+        TODO("Not yet implemented")
+    }
+
+    fun onRadioButtonClicked(view: View) {
```

Comment:
Sao không để luôn n là RadioButton, đỡ phải cast bên dưới
```suggestion
    fun onRadioButtonClicked(radioButton: RadioButton) {
```